recipes/correct-ts-specifiers/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "correct-ts-specifiers",
-  "version": "1.0.0-rc.1",
+  "name": "@nodejs/correct-ts-specifiers",
+  "version": "1.0.0",
   "description": "Replace erroneous 'js' or omitted file extensions of import specifiers in TypeScript files.",
   "type": "module",
   "main": "./src/workflow.ts",
   "engines": {
-    "node": ">=22.6.0"
+    "node": ">=22.15.0"
```

Comment:
v20 can't be supported ?

Reply:
registerHooks isn't backported to it yet. this tool is run on a dev's machine, not in production or CI, so it's not as critical to support older versions of node.